python/artm/model.py

```diff
@@ -653,7 +658,7 @@ def get_phi(self, topic_names=None, class_ids=None):
 
         return phi_data_frame
 
-    def fit_transform(self, topic_names=None, remove_theta=False):
+    def get_theta(self, topic_names=None, remove_theta=False, use_doc_titles=False):
```

Comment:
I'd keep old method too for now (ideally tagged with `obsolete`, but I don't know how to do this in python)
```
def fit_transform(self, topic_names=None, remove_theta=False):
  return self.get_theta(topic_names, remove_theta);
```